js/module.ts

```diff
@@ -1185,6 +1185,12 @@ export interface ISource extends IConfigurable, IReleasable {
      * Easy way to disable a filter.
      */
     enabled: boolean;
+	
+    /**
+     * Function to get latest version of settings
+	 * Expensive, shouldn't be used unless sure
+     */
+    slowUncachedSettings(): ISettings;
```

Comment:
I notice this is a function - should it be a property like the existing `settings` property?

i.e. it would be `readonly slowUncachedSettings: ISettings;`